packages/runtime/src/useCases/transport/relationshipTemplates/CreateTokenForOwnRelationshipTemplate.ts

```diff
@@ -42,7 +42,8 @@ export class CreateTokenForOwnTemplateUseCase extends UseCase<CreateTokenForOwnT
 
         const tokenContent = TokenContentRelationshipTemplate.from({
             templateId: template.id,
-            secretKey: template.secretKey
+            secretKey: template.secretKey,
+            forIdentity: template.cache?.forIdentity
```

Comment:
we should validate that this forIdentity matches the forIdentity of the Token, same for other CreateTokenFor*UseCases

Reply:
I'd rather take the personalization from the Template rather than validating a personalization given from the user

Reply:
Yeah, if there is a personalization. But what you could do is having an unpersonalized template and putting a personalization on with a token.